person.rb

```diff
@@ -1,6 +1,6 @@
 require_relative 'nameable'
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- Hey, the rentals and book should be a getter and setter variable, so kindly make it accessible with `attr_accessor`

`attr_accessor :name, :age, :books, :rentals`